src/Components/FeatherIcons.rei

```diff
@@ -0,0 +1,4 @@
+let folder: SVGIcon.t;
+let codeSandbox: SVGIcon.t;
```

Comment:
I realized this was the branded codesandbox icon (https://codesandbox.io/) - I worry this would be confusing. 

We could try the package icon instead:
https://github.com/feathericons/feather/blob/master/icons/package.svg

Reply:
(CodeSandbox sure has a cool logo, though!)

Reply:
Oh that's my bad, I didn't realize `codesandbox` was a proper noun 🤦 -- I'll change that out for the package one now!

Reply:
New icon looks great :+1: